functions/run.sh

```diff
@@ -3,8 +3,9 @@
 source ./util.sh
 source ./init.sh $CLUSTER
 
-dir=`dirname "${BASH_SOURCE[0]}"`
-
-for test in uppercase; do
+# TODO: move hello out of functions
```

Comment:
postponing this for now, as I have several questions as I'm not sure what a good layout could be
are there different eventing scenarios we want to support in the future?

Reply:
I think there will be multiple eventing scenarios in time, particularly around different bus capabilities, like the ability to recover from errors.